test/test_unistd.rs

```diff
@@ -600,6 +600,62 @@ fn test_symlinkat() {
     );
 }
 
+#[test]
+fn test_getpwuid() {
+    let res = User::query( UserQuery::Uid(Uid::from_raw(1)) ).unwrap();
```

Comment:
Are you sure that there will be a UID 1?

Reply:
As far I can tell, yes.

Reply:
It might be better to use UID (and GID) 0, which is always guaranteed to be the superuser, and on almost all Unix systems will have the name `root`.

Reply:
Did it.